google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -426,13 +426,3 @@ account.plugin.removal.requires.restart.title=Google Cloud Tools: Please restart
 account.plugin.removal.requires.restart.text=The Google Account plugin is no longer required and \
   has been uninstalled.  Please restart your IDE for it to take effect.  If you cancel you will \
   experience problems using the Google Cloud Tools plugin. \n\nOK to restart?
-
-cloud.project.selector.signin.message=Sign in with your Google account to list your Google Developers Console projects.
-cloud.project.selector.open.dialog.tooltip=Opens a dialog to select a Google Cloud project
-cloud.project.selector.no.selected.project=Select a Google Cloud project
-cloud.project.selector.not.signed.in=Sign in with your Google account
-cloud.project.selector.dialog.title=Select Google Cloud Project
-cloud.project.selector.project.list.project.name.column=Project Name
-cloud.project.selector.project.list.project.id.column=Project ID
-project.selector.loader.couldnotconnect=Could not connect to Google. Please make sure you have a working internet connection.
```

Comment:
was this not used before? I see it is removed here, but not added to the new bundle